anheumai-dparys/context/context_test.go

```diff
@@ -0,0 +1,97 @@
+package ctx_is_king
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+type SpyStore struct {
+	response string
+	t        *testing.T
+}
+
+func (s *SpyStore) Fetch(ctx context.Context) (string, error) {
+	data := make(chan string, 1)
+
+	go func() {
+		var result string
+		for _, c := range s.response {
```

Comment:
What is c? a single context? I haven't done that chapter yet.

Reply:
no, we're iterating here over a string character by character, c == character

Reply:
Thank you very much for clarification. I really appreciate this :-)